samples/tacos/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2023 Slack Technologies, LLC
+// SPDX-License-Identifier: Apache-2.0
+import org.jetbrains.kotlin.gradle.internal.KaptGenerateStubsTask
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+  id("com.android.application")
+  kotlin("android")
+  kotlin("kapt")
+  kotlin("plugin.parcelize")
+  alias(libs.plugins.ksp)
+}
+
+android {
+  namespace = "com.slack.circuit.tacos"
+
+  defaultConfig {
+    minSdk = 33
+    testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+  }
+
+  testOptions { unitTests.isIncludeAndroidResources = true }
+}
+
+tasks
+  .withType<KotlinCompile>()
+  .matching { it !is KaptGenerateStubsTask }
+  .configureEach {
+    compilerOptions {
+      freeCompilerArgs.addAll(
+        "-opt-in=androidx.compose.material.ExperimentalMaterialApi",
```

Comment:
Do we use M2 anymore anywhere?

Reply:
Good catch; it can be removed ([24f2538](https://github.com/slackhq/circuit/pull/496/commits/24f2538f17345b56772263b114aa1b5bc4aa5935)). I started by copying the star app build file and mustn't have looked that closely.